x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/rules_table/utils.ts

```diff
@@ -31,6 +31,7 @@ export function mergeRules(currentRules: Rule[], newRules: Rule[]): Rule[] {
  * @param sortingOptions SortingOptions
  */
 export function getRulesComparator(sortingOptions: SortingOptions) {
+  // eslint-disable-next-line complexity
```

Comment:
This function is inherently complex, though can refactor a few of the `case`'s to separate functions to get below the magic `20` number.

Reply:
I think it's OK to have the complexity check disabled for this function. Extracting some of its parts will unlikely make it less complex.